demo/app.js

```diff
@@ -502,10 +523,37 @@ const Demo = React.createClass({
     });
   },
 
+  _handleNextSiblingClick () {
+  },
```

Comment:
@cerinman I removed the console logs so we don't have eslint errors, but kept these in because they are required handlers in order to display the nav section with animation that I wanted to demonstrate.